README.md

```diff
@@ -12,219 +12,249 @@
 
-Learning Semantic Representations Using Convolutional Neural Networks for Web Search (2014a).
-[website](https://www.microsoft.com/en-us/research/publication/learning-semantic-representations-using-convolutional-neural-networks-for-web-search/) | [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/www2014_cdssm_p07.pdf)
+* [**Gov2**](https://trec.nist.gov/data/terabyte.html) is a large Web collection where the pages are crawled from .gov. It consists of 25 million documents in total. The queries are accumulated over TREC Terabyte Tracks 2004, 2005, and 2006. There are 150 queries in total.
 
-A Latent Semantic Model with Convolutional-Pooling Structure for Information Retrieval (2014b).
-[website](https://www.microsoft.com/en-us/research/publication/a-latent-semantic-model-with-convolutional-pooling-structure-for-information-retrieval/) | [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/cikm2014_cdssm_final.pdf)
+* [**MSMARCO Passage Reranking**](http://www.msmarco.org/dataset.aspx) provides a large number of information question-style queries from Bing's search logs. There passages are annotated by humans with relevant/non-relevant labels. There are 8,841822 documents in total. There are 6,980 queries and 48,598 queries for validation and test, respectively.
 
-#### LSTM-DSSM
-![status](artworks/not-in-plan.svg)
+Neural information retrieval (NeuIR) models include:
 
-Deep Sentence Embedding Using Long Short-Term Memory Networks: Analysis and Application to Information Retrieval.
-[website](https://arxiv.org/abs/1502.06922) | [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/02/LSTM_DSSM_IEEE_TASLP.pdf)
+1. **DSSM** [[code]](https://github.com/NTMC-Community/MatchZoo/blob/master/matchzoo/models/dssm.py) [[pdf]](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/cikm2013_DSSM_fullversion.pdf) ![status](artworks/ready.svg)
 
-#### ARC-I
-![status](artworks/ready.svg)
+   Learning Deep Structured Semantic Models for Web Search using Clickthrough Data. *CIKM 2013*.
+   [website](https://www.microsoft.com/en-us/research/publication/learning-deep-structured-semantic-models-for-web-search-using-clickthrough-data/) | [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/cikm2013_DSSM_fullversion.pdf) | [tutorial](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/07/dl-summer-school-2017.-Jianfeng-Gao.v2.pdf)
 
-Convolutional Neural Network Architectures for Matching Natural Language Sentences.
-[website](https://arxiv.org/abs/1503.03244) | [paper](https://arxiv.org/pdf/1503.03244.pdf)
+2. **CDSSM**  [[code]](https://github.com/NTMC-Community/MatchZoo/blob/master/matchzoo/models/cdssm.py) [[pdf]](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/www2014_cdssm_p07.pdf) ![status](artworks/ready.svg)
```

Comment:
Duplicate link in `[pdf]` and `paper`. 
`[code] [paper] [website]` is better.

Reply:
Agree. pdf and paper are duplicated. There are some missing links in the pdf field.

Reply:
It's really duplicate. It will be removed.